src/adapters/Cornerstone/Bidirectional.js

```diff
@@ -133,10 +134,10 @@ class Bidirectional {
             toolName: "Bidirectional",
             visible: true,
             finding: findingGroup
-                ? findingGroup.ConceptCodeSequence
+                ? addAccessors(findingGroup.ConceptCodeSequence)
```

Comment:
Can't you use the same way we get the `defaultState` in the other tools?

Reply:
The default state doesn't seem to always include the finding/findingSite data - probably I should revert just this single change, but otherwise leave setting the finding /site data to another issue.